### PR TITLE
V0.8

### PR DESCRIPTION
**Pull request by bruAristimunha**

### CODE REVIEW

**Comment by robintibor** [comment]
Guess you're gonna have to remove scale in quite a few places

File: test/unit_tests/preprocessing/test_preprocess.py [hunk 1]
```diff
@@ -117,18 +117,12 @@ def test_preprocess_windows_callable_on_object(windows_concat_ds):
...
-    with pytest.warns(FutureWarning, match=msg):
-        deprecated_scale(np.random.rand(2, 2), factor=2)
-
 
 def test_scale_continuous(base_concat_ds):
     factor = 1e6
     preprocessors = [
         Preprocessor('pick_types', eeg=True, meg=False, stim=False),
-        Preprocessor(deprecated_scale, factor=factor)
+        Preprocessor(lambda data: multiply(data, factor), factor=factor)
```
Comment (robintibor) [inline test/unit_tests/preprocessing/test_preprocess.py:125]:
```suggestion
        Preprocessor(multiply, factor=factor)
```
Would this also work, if yes, would be nicer, but don't know if this works @bruAristimunha ?

Comment (bruAristimunha) [reply test/unit_tests/preprocessing/test_preprocess.py]: I liked your suggestion @robintibor, but it didn't work

Comment (robintibor) [reply test/unit_tests/preprocessing/test_preprocess.py]: ok pity

**Pull request merged**